dotCMS/src/main/java/com/dotmarketing/portlets/files/action/UploadMultipleFilesAction.java

```diff
@@ -238,7 +243,7 @@ public void _saveFileAsset(ActionRequest req, ActionResponse res,PortletConfig c
 
 				if(!APILocator.getFolderAPI().matchFilter(folder,fileName))
 	            {
-				   customMessage += fileName + ", ";
+				   customMessage += "<strong>" + fileName + "</strong>, ";
```

Comment:
Codacy found an issue: [Prefer StringBuilder (non-synchronized) or StringBuffer (synchronized) over += for concatenating strings](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6250124)